microsetta_interface/implementation.py

```diff
@@ -2768,14 +2769,6 @@ def post_claim_samples(*, account_id=None, source_id=None, body=None,
     if has_error:
         return survey_output
 
-    # TODO: this will have to get more nuanced when we add animal surveys?
```

Comment:
I think there are still some pieces that can come out of this function, e.g. the survey_output section directly above (GitHub's UI won't let me contextually comment on lines that aren't being changed in the PR). Also, the comment above the function definition is no longer applicable.

Reply:
Ah ok, so all we're left here now is just the reconsent if needed and claiming of a sample https://github.com/biocore/microsetta-interface/blob/fba3472ccff20692c92dbaed230fe7977e1905df/microsetta_interface/implementation.py#L2769-L2776